internal/provider/features_test.go

```diff
@@ -183,7 +183,7 @@ func TestExpandFeatures(t *testing.T) {
 						map[string]interface{}{
 							"detach_implicit_data_disk_on_deletion": true,
 							"delete_os_disk_on_deletion":            true,
-							"graceful_shutdown":                     true,
+							"graceful_shutdown":                     true, // TODO: Remove in 5.0
```

Comment:
There's no point testing this any more since it doesn't do anything?
```suggestion
```

Reply:
Fixed.